lib/models/infrastructure.rb

```diff
@@ -65,36 +72,32 @@ def infrastructure_totals
       totals
     end
   end
-  # attr_reader :total_storage_gb
-  # attr_reader :total_disk_io_kilobytes_per_second
-  # attr_reader :total_lan_io_kilobits_per_second
-  # attr_reader :total_wan_io_kilobits_per_second
-  # attr_reader :vcpu_per_machine
-  # attr_reader :machines_per_core
 
+  # TODO CHECK IF WE STILL NEED THIS METHOD
   def enabled?
-    # Assumes 1:1 meter:infrastructure ratio
-    self.meter_instance.enabled
+    self.enabled
   end
 
   def disable
-    self.meter_instance.update_attribute('enabled', false)
+    self.update_attribute('enabled', false)
   end
+
   def enable
-    self.meter_instance.update_attribute('enabled', true)
+    self.update_attribute('enabled', true)
   end
 
-  def submit_create(infrastructure_endpoint)
+  def submit_create
     response = nil
     begin
       logger.info "Submitting #{name} to API for creation in UC6"
       self.tags = name if tags.blank?
-      response = hyper_client.post(infrastructure_endpoint, api_format)
-
+      response = hyper_client.post(infrastructures_post_url, api_format)
       if ( response and response.code == 200 )
-        self.remote_id = response.remote_id
-        self.meter_instance = MeterInstance.find_or_create_in_uc6(infrastructure_name: name,
-                                                                  infrastructure_endpoint: "#{infrastructure_endpoint}/#{remote_id}")
+        self.remote_id = JSON.parse(response)["id"]
```

Comment:
I would use single quotes for `id` since there is no variable interpolation within the string of that key